### ISSUE TITLE
"Download propers: no" setting not respected.

### ISSUE THREAD

**Issue by Crowley007** [Radarr/Radarr#546]
A "Proper" version of a movie gets downloaded every time, and Radarr ignores preferred words completely if a "proper" is available.

Only way around that was to set a required word tag to get a better quality release.

**Comment by onedr0p** [comment]
I see this in the code, it appears to be the same as Sonarr... hm..

```csharp
        public virtual Decision IsSatisfiedBy(RemoteMovie subject, SearchCriteriaBase searchCriteria)
        {
            if (searchCriteria != null)
            {
                return Decision.Accept();
            }

            if (subject.Movie.MovieFile.Value == null)
            {
                return Decision.Accept();
            }

            var file = subject.Movie.MovieFile.Value;

                if (_qualityUpgradableSpecification.IsRevisionUpgrade(file.Quality, subject.ParsedMovieInfo.Quality))
                {
                    if (file.DateAdded < DateTime.Today.AddDays(-7))
                    {
                        _logger.Debug("Proper for old file, rejecting: {0}", subject);
                        return Decision.Reject("Proper for old file");
                    }

                    if (!_configService.AutoDownloadPropers)
                    {
                        _logger.Debug("Auto downloading of propers is disabled");
                        return Decision.Reject("Proper downloading is disabled");
                    }
                }
            

            return Decision.Accept();
        }
```

**Comment by onedr0p** [comment]
Just so we're on the same page: Proper releases will still get downloaded if it matches a movie, that setting is to only to allow upgrading releases to proper/repack if they are picked up by rss sync. This is working as intended as far as I can tell.

**Comment by Crowley007** [comment]
It's still a problem, a "Proper" will get downloaded even if there are much better quality releases available. 

When I posted this, a movie had several 15Gb and near 30GB Remux releases, but the automatic search picked lower tan 10gb "Proper" release every time, and ignored every preferred word. For Now I have forced Radarr to download only Remuxes so it's not a problem, but it sound weird for intended behavior.
